### PR TITLE
Feature/mp/begrepbase

### PR DESCRIPTION
**Pull request by markkopaas**
Jira issue link: [link](LINK_HERE)

> check applicable boxes

- [ ] I have made tests to match the user stories
- [ ] This code does not require tests that match user stories

### CODE REVIEW

File: applications/nginx-search/src/main/docker/nginx.conf [hunk 1]
```diff
@@ -102,6 +102,11 @@ http {
             proxy_pass http://api-cat:8080;
         }
 
+        location /concept-cat {
```
Comment (dskogan) [inline applications/nginx-search/src/main/docker/nginx.conf]: Why not use /concepts here. It is the logical path!

File: applications/concept-cat/src/main/java/no/ccat/controller/ConceptGetController.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    @Autowired
+    public ConceptGetController(ConceptDenormalizedRepository conceptDenormalizedRepository) {
+        this.conceptDenormalizedRepository = conceptDenormalizedRepository;
+    }
+
+    @ApiOperation(value = "Get a specific concept", response = ConceptDenormalized.class)
+    @RequestMapping(value = "/{id}", method = RequestMethod.GET, produces = "application/json")
+    public ConceptDenormalized getConceptDenormalized(@PathVariable String id) throws NotFoundException {
+        logger.info("request for {}", id);
```
Comment (thomasnokleby) [inline applications/concept-cat/src/main/java/no/ccat/controller/ConceptGetController.java]: Er dette rett logging-nivå ?

File: applications/concept-cat/src/main/java/no/ccat/controller/ConceptSearchController.java [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@CrossOrigin
+@RestController
+@RequestMapping(value = "/concepts")
+public class ConceptSearchController {
+    public static final String MISSING = "MISSING";
+    public static final int MAX_AGGREGATIONS = 10000;
```
Comment (thomasnokleby) [inline applications/concept-cat/src/main/java/no/ccat/controller/ConceptSearchController.java]: Sikkert riktig, men hvorfor 10000 ?

File: applications/nginx-search/src/main/docker/nginx.conf (hunk 1 above)
Comment (markkopaas) [reply applications/nginx-search/src/main/docker/nginx.conf]: god idé! takk!

File: applications/concept-cat/src/main/java/no/ccat/controller/ConceptGetController.java (hunk 2 above)
Comment (markkopaas) [reply applications/concept-cat/src/main/java/no/ccat/controller/ConceptGetController.java]: jeg tror vi må bruke debug, takk!

File: applications/concept-cat/src/main/java/no/ccat/controller/ConceptSearchController.java (hunk 3 above)
Comment (markkopaas) [reply applications/concept-cat/src/main/java/no/ccat/controller/ConceptSearchController.java]: Integer.MAX_VALUE ser mer beskrivelig

**Pull request merged**